octoprint_v8theme/static/js/v8theme.js

```diff
@@ -1,7 +1,157 @@
+                },
+                points: {
+                    show: false
+                }
+            },
+            yaxis: {
+                min: 0
+            },
+            xaxis: {
+                mode: "time",
+                ticks: 10,
+                minTickSize: [2, "minute"],
+                tickFormatter: function(val, axis) {
+                    if (val == undefined || val == 0)
+                        return ""; // we don't want to display the minutes since the epoch if not connected yet ;)
+
+                    // current time in milliseconds in UTC
+                    var timestampUtc = Date.now();
+
+                    // calculate difference in milliseconds
+                    var diff = timestampUtc - val;
+
+                    // convert to minutes
+                    var diffInMins = Math.round(diff / (60 * 1000));
+                    if (diffInMins == 0)
+                        return gettext("just now!");
+                    else
+                        return "- " + diffInMins + " " + gettext("min");
+                }
+            },
+            colors: Colors,
+            shadowSize: 1,
+            tooltip: true,
+            //activate tooltip
+            tooltipOpts: {
+                content: "%s : %y.0",
+                shifts: {
+                    x: -30,
+                    y: -50
+                },
+                defaultTheme: false
+            }
+        };
+        
+        self.parseCustomControls = function() {
+            $("#control .custom_section").each(function() {
+                var accordionName = $('h1 span', this).text();
+                accordionName_nospace = accordionName.replace(/\s+/g, '');
+                $("#control_wrapper").after("<div id='" + accordionName_nospace + "_wrapper' class='accordion-group'><div class='accordion-heading'><a class='accordion-toggle' data-toggle='collapse' data-target='#" + accordionName_nospace + "_main'><i class='icon-info-sign'></i> " + accordionName + " </a></div><div id='" + accordionName_nospace + "_main' class='accordion-body collapse in '><div class='accordion-inner'></div></div>");
+                if ($(".custom_section_horizontal", this).length) {
+                    $(".custom_section_horizontal", this).appendTo("#" + accordionName_nospace + "_main .accordion-inner");
+                } else if ($(".custom_section_vertical", this).length) {
+                    $(".custom_section_vertical", this).appendTo("#" + accordionName_nospace + "_main .accordion-inner");
+                } else if ($(".custom_section_horizontal_grid", this).length) {
+                    $(".custom_section_horizontal_grid", this).appendTo("#" + accordionName_nospace + "_main .accordion-inner");
+                }
+                $(this).remove();
+            });
+        };
+
+        self.temperature.updatePlot = function() {
+            var graph = $("#temperature-graph");
+            if (graph.length) {
```

Comment:
Good point and I was thinking the same thing when looking it over but again this is from OctoPrint source. Should I go ahead and change?